client/landing/stepper/declarative-flow/entrepreneur-flow.ts

```diff
@@ -58,8 +58,7 @@ const entrepreneurFlow: Flow = {
 			}
 
 			const redirectTarget =
-				`/setup/entrepreneur/create-site` +
-				( hasFlowParams ? encodeURIComponent( '?' + flowParams.toString() ) : '' );
+				`/setup/entrepreneur/create-site` + ( hasFlowParams ? '?' + flowParams.toString() : '' );
```

Comment:
Explanation: getLoginUrl() is already encoding the URL so we don't have to encode it here.